src/main/java/org/jitsi/jigasi/transcription/TranscriptionAlternative.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Jigasi, the JItsi GAteway to SIP.
+ *
+ * Copyright @ 2017 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jigasi.transcription;
+
+/**
+ * An alternative transcription of a chunk of audio.
+ *
+ * @author Boris Grozev
+ */
+public class TranscriptionAlternative
+{
+    /**
+     * The default confidence value.
+     */
+    private static double DEFAULT_CONFIDENCE = 1.0D;
```

Comment:
This constant was meant as a default when the confidence or a result is not known (e.g. if another s2t API is used)